doc/index.rst

```diff
@@ -233,6 +233,7 @@ PennyLane is **free** and **open source**, released under the Apache License, Ve
    code/qml_measurements
    code/qml_pytrees
    code/qml_operation
+   code/qml_decomposition
```

Comment:
Is this going to be internals, part of the user API?

Reply:
I guess this would be user facing. I'll move it.